src/main/kotlin/ink/abb/pogo/scraper/tasks/EvolvePokemon.kt

```diff
@@ -61,7 +61,11 @@ class EvolvePokemon : Task {
                     if (bot.api.inventories.candyjar.getCandies(pokemonMeta.family) >= pokemonMeta.candyToEvolve) {
                         Log.yellow("Evolving ${it.pokemonId.name} CP ${it.cp} IV ${it.getIvPercentage()}%")
                         val evolveResult = it.evolve()
-                        Thread.sleep(300)
+                        if( settings.evolveTimeDelay > 300){
```

Comment:
There was a bug issue, and someone fixed it with this time delay. The error was, when stack evolution was reached, there was a null exceptions. If you dont believe me, just look where the old line (Thread.sleep(300)) came from, and the issue will probably be linked to the PR